io.github.chen08209.FlClash.yml

```diff
@@ -0,0 +1,45 @@
+id: io.github.chen08209.FlClash
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: AppRun
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=com.follow.clash
+
+modules:
+  - name: FlClash
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ./io.github.chen08209.FlClash.metainfo.xml ${FLATPAK_DEST}/share/metainfo/io.github.chen08209.FlClash.metainfo.xml
+
+      - chmod +x ./FlClash.AppImage
+      - ./FlClash.AppImage --appimage-extract
+
+      - sed -i 's/LD_LIBRARY_PATH=usr\/lib FlClash %u/AppRun %U/g' ./squashfs-root/FlClash.desktop
```

Comment:
LD_LIBRARY_PATH is not supposed to be set in desktop file. Use your own wrapper script and override the Exec key of desktop file with desktop-file-edit.

Reply:
The LD_LIBRARY_PATH env come from upstream, I replace it by using `sed`.

Reply:
use desktop-file-edit